frontend-api/build.gradle

```diff
@@ -36,7 +36,8 @@ dependencies {
             configurations.s3,
             configurations.kms,
             configurations.dynamodb,
-            configurations.pact_consumer
+            configurations.pact_consumer,
+            'org.wiremock:wiremock-jetty12:3.10.0'
```

Comment:
We should keep an eye on test suite run times if we're introducing wiremock. In my experience it can slow things down quite a lot if not used carefully

Reply:
Some local testing showed a small but noticeable increase in time, negligible compared to integration test execution times but still there.  I made a small change to start the wiremock server before all the tests and only shut it down after all the tests have completed which made the text execution times broadly the same as before wiremock was introduced.  Will keep this under review.